profiles/standard/standard.install

```diff
@@ -106,12 +106,23 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
-  // Set front page to "node".
-  config_set('system.core', 'site_frontpage', 'node');
+  // Set front page to "home".
+  config_set('system.core', 'site_frontpage', 'home');
+  backdrop_flush_all_caches();
 
-  // Insert default pre-defined node types into the database. For a complete
-  // list of available node type attributes, refer to the node type API
-  // documentation at: http://api.drupal.org/api/HEAD/function/hook_node_info.
+  // Set front page title to include sitename.
+  $settings = array(
+    'title' => t('Welcome to @site', array('@site' => config_get('system.core', 'site_name'))),
+    'title_display' => 'custom',
+    'title_block' => NULL,
+  );
+  $home_config = config('layout.layout.home');
+  $home_config->set('settings', $settings);
+  $home_config->save();
+
+  // Create default pre-defined node types. For a complete list of available
+  // node type attributes, refer to the node type API documentation at:
+  // http://api.drupal.org/api/HEAD/function/hook_node_info.
```

Comment:
Again, still have a reference to `api.drupal.org` here for some reason. Should be `api.backdropcms.org` or removed.